spec/helpers/federal_register_documents_helper_spec.rb

```diff
@@ -2,7 +2,28 @@
 
 describe FederalRegisterDocumentsHelper do
   describe '#federal_register_document_comment_period' do
-    let(:document) { mock_model(FederalRegisterDocument) }
+    let(:document) do
+      mock_model(
+        FederalRegisterDocument,
+        document_type: "Notice",
+        publication_date: "Mon, 09 Jun 2014".to_date,
+        contributing_agency_names: ['Internal Revenue Service',
+                                    'International Trade Administration',
+                                    'National Oceanic and Atmospheric Administration']
+      )
+    end
+
+    context 'when federal_register_document_info is called' do
+      let(:result) do
+        "A <span>Notice</span> by the <span>Internal " \
```

Comment:
This would be cleaner as a heredoc:
```ruby
      let(:result) do
        <<~HTML
          A <span>Notice</span> by the <span>Internal
          Revenue Service</span>, the <span>International
          Trade Administration</span> and the <span>National
          Oceanic and Atmospheric Administration</span>
          posted on <span>June 9, 2014</span>.
        HTML
      end
```